### PR TITLE
Make a backup at install time only if it is not there yet

### PR DESCRIPTION
**Pull request by Etn40ff**
Currently if one installs kernelctl twice with the same version of codex `configure` will fail. 

This PR fixes this behaviour

### CODE REVIEW

**Comment by Eeems** [comment]
Just an FYI, but the build will not create new packages if the version number doesn't change.

**Review by Eeems — CHANGES_REQUESTED** [review]
image

**Comment by Eeems** [comment]
Running the script manually produces the following:
image

File: package/kernelctl/package [hunk 1]
```diff
@@ -27,5 +27,15 @@ package() {
 }
 
 configure() {
-    kernelctl backup vanilla
+    if [[ ! -e "/opt/usr/share/kernelctl/vanilla-$(< /etc/version)" ]]; then
```
Comment (Eeems) [inline package/kernelctl/package]:
```suggestion
    if [[ "$(kernelctl list | tail -n +2 | awk '{print $2}' | grep "vanilla-$(< /etc/version)")" == "" ]]; then
```

Comment (Etn40ff) [reply package/kernelctl/package]: LGTM, sorry for the omission

**Review by Eeems — APPROVED** [review]

**Pull request merged**